packages/next-templates/src/app/globals.css

```diff
@@ -215,6 +215,7 @@ utrecht-alert
   padding-inline: var(--voorbeeld-space-block-cat);
   max-inline-size: calc(var(--utrecht-article-max-inline-size) * 1.125);
   inline-size: 100%;
```

Comment:
haal alles wat je hebt toegevoegd weg en dan deze inline size ook weg en met weg bedoel ik alles in het groen moet deleted zijn kijk dan of de size goed is